src/main/java/com/flexrate/flexrate_back/member/api/WebAuthnService.java

```diff
@@ -0,0 +1,48 @@
+package com.flexrate.flexrate_back.member.api;
+
+import com.flexrate.flexrate_back.auth.domain.FidoCredential;
+import com.flexrate.flexrate_back.common.exception.ErrorCode;
+import com.flexrate.flexrate_back.common.exception.FlexrateException;
+import com.flexrate.flexrate_back.member.domain.Member;
+import com.flexrate.flexrate_back.member.domain.repository.FidoCredentialRepository;
+import com.flexrate.flexrate_back.member.domain.repository.MemberRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+public class WebAuthnService {
+
+    private final MemberRepository memberRepository;
+    private final FidoCredentialRepository fidoCredentialRepository;
+
+    public WebAuthnService(MemberRepository memberRepository, FidoCredentialRepository fidoCredentialRepository) {
+        this.memberRepository = memberRepository;
+        this.fidoCredentialRepository = fidoCredentialRepository;
+    }
```

Comment:
전에도 말씀드렸었던 부분인데, 저희 프로젝트는 `@RequiredArgsConstructor`을 사용하고있어요. 
이 어노테이션을 사용하고, 생성자는 제거해주세요.
코드 작성하실때는, 항상 기존 코드를 옆에 띄워두고 컨벤션을 맞춰가며 작업해주세요~
아니면 AI 프롬프트 작성하실때 기존 코드를 같이 넣어서 컨벤션이 맞는지 검토하는 과정도 추가하는걸 추천드려요.

```suggestion
@Service
@RequiredArgsConstructor
public class WebAuthnService {

    private final MemberRepository memberRepository;
    private final FidoCredentialRepository fidoCredentialRepository;
```